app/controllers/index.js

```diff
@@ -16,10 +16,6 @@ export default class IndexController extends Controller {
   @tracked showUserStateModal = false;
   @tracked newStatus;
 
-  get isDevMode() {
-    return this.featureFlag.isDevMode;
```

Comment:
Line 13 can also be removed I think.

Reply:
Thanku for pointing it out , I have removed it. Can you please approve now?